src/main/java/ladder/domain/Person.java

```diff
@@ -0,0 +1,36 @@
+package ladder.domain;
+
+import ladder.util.ErrorMessage;
+
+public class Person {
+    private static final int MAX_NAME_LENGTH = 5;
+
+    private String name;
+
+    public Person(String name) {
+        checkName(name);
+
+        this.name = name;
+    }
+
+    private void checkName(final String name) {
+        checkEmpty(name);
+        checkNameLength(name);
+    }
+
+    private void checkEmpty(final String names) {
+        if (names == null) {
```

Comment:
```suggestion
        if (Objects.isNull(names)) {
```
Objects# 클래스를 활용하여 가독성을 높일수 있으니 사용해보세요 !